examples/cluster

```diff
@@ -0,0 +1,53 @@
+// This is environment description file for starting up a multinode CM-Well cluster. 
+// Given 10 nodes with hostnames cmw1, cmw2, ... cmw10 
+// each having 24GB of RAM and 6 storage drives mounted at /mnt/d1 .. /mnt/d6
+// which all have the user "cmuser", and have SSH trust with each other. 
+// Note: a sudoer is needed for first time installation.
+// Save this file as "cluster", and once you've built CM-Well, run CM-Well Console:
+// server/cmwell-cons/app/cmwell.sh
+//
+// within CM-Well Console, type the following:
+// :load cluster
+// cluster.prepareMachines()
+// cluster.install
+
+val hostnames = (1 to 10).map("cmw" + _)
```

Comment:
Suggest to create a "stupid" list like: 
val hostnames = List("10.10.10.1", "10.10.10.2", "10.10.10.3" etc )